Doc 3078 Database section of RC - updates from Doc Bash (#2922)
RedisLabs/redislabs-docs: content/rc/databases/configuration/clustering.md

    and pipelining are supported with Redis Cloud cluster
    like if it were a non-cluster DB.

> ## Managing the hashing policy

Reviewer: Relrefs that previously pointed to `#changing-the-hashing-policy` will also need to be updated unless you want to create a custom alias to match the old heading.
```suggestion
## Manage the hashing policy
```
OR
```suggestion
## Manage the hashing policy {#changing-the-hashing-policy}
```

> Redis defaults to the [Standard hashing policy]({{< relref "/rs/databases/durability-ha/clustering#standard-hashing-policy" >}}). 

Reviewer: ```suggestion
Redis defaults to the [standard hashing policy]({{< relref "/rs/databases/durability-ha/clustering#standard-hashing-policy" >}}). 
```

You can use the '{...}' pattern to direct related keys to the same hash
slot, so that multi-key operations are supported on them. On the other
hand, not using a hashtag in the key's name results in a

> (statistically) even distribution of keys across the keyspace's shards 
> which results in better resource utilization. 

Reviewer: ```suggestion
(statistically) even distribution of keys across the keyspace's shards, 
which improves resource utilization. 
```
